src/bundle/Resources/public/js/scripts/helpers/tooltips.helper.js

```diff
@@ -1,35 +1,39 @@
 (function(global, doc, eZ, $) {
     const TOOLTIPS_SELECTOR = '[title]';
     const parse = (baseElement = doc) => {
-        const tooltipNodes = baseElement.querySelectorAll(TOOLTIPS_SELECTOR);
+        if (baseElement) {
```

Comment:
I wonder if it wouldnt be better to do reverse if
```
if (!baseElement) {
  return;
}
```
thanks to it you would avoid adding extra tabs in next lines and it would be much shorter diff